Automattic/woocommerce-services
# Hide Jetpack Banner when Using WCS Banner

In this PR:

- Remove jetpack's banners from the `admin_notices` hook if we are adding the WCS connect banner
- Delete old jetpack connect and tos notice code from the plugin

To Test:
- Start with a setup where Jetpack is not connected
- Make sure you're not using `JETPACK_DEV_DEBUG` in your config
- Go the plugins page and notice that the WCS connect banner is showing without the Jetpack banner
- Disable the WCS plugin, notice that the jetpack banner now shows
- Make sure nothing in the plugin is broken

@jeffstieler @v18 @DanReyLop @marcinbot @olaolusoga
